jax/interpreters/partial_eval.py

```diff
@@ -785,88 +798,65 @@ def convert_envvars_to_constvars(jaxpr: Jaxpr, num_env_vars: int) -> Jaxpr:
 def _split_aval(unknown: bool, aval: AbstractValue) -> Tuple[AbstractValue, AbstractValue]:
   return (abstract_unit, aval) if unknown else (aval, abstract_unit)
 
-def partial_eval_jaxpr(jaxpr: ClosedJaxpr, unknowns: Sequence[bool],
-                       instantiate: Union[bool, Sequence[bool]],
-                       ) -> Tuple[ClosedJaxpr, ClosedJaxpr, Sequence[bool]]:
-  """Specializes a Jaxpr given an indication of which inputs are known.
```

Comment:
Keep (some of) this docstring around?

Reply:
Well, I'd rather not document the deprecated one, and most of it is incorrect for the new one. I can add a docstring for the new function though...

Reply:
Done!

Reply:
Nice. I've found it to be one of the more useful comments in this file, because of the example.

Reply:
Good suggestion.